tensorflow/contrib/learn/python/learn/models.py

```diff
@@ -153,7 +153,7 @@ def logistic_regression(x,
                                   bias)
     # If no class weight provided, try to retrieve one from pre-defined
     # tensor name in the graph.
-    if not class_weight:
+    if class_weight is not None:
```

Comment:
this reverses the intent of the original code and attempts to reassign the class_weight tensor if it exists.